Makefile_util.am

```diff
@@ -38,3 +38,39 @@ fuzz/fuzz_ossfuzz: fuzz/fuzz-fuzz.o libass/libass.la $(srcdir)/libass.pc
 		$$(pkg-config --static --libs $(srcdir)/libass.pc | sed -e 's/-lm //g' -e 's/-lass //g') \
 		-Wl,-Bdynamic
 endif
+
+
+if ENABLE_CHECKASM
+
+check_PROGRAMS += checkasm/checkasm
+
+.asm.o:
+	$(nasm_verbose)$(AS) $(ASFLAGS) -I$(top_srcdir)/libass/ -o $@ $<
+
+checkasm_checkasm_SOURCES = \
+    checkasm/blend_bitmaps.c \
+    checkasm/be_blur.c \
+    checkasm/checkasm.h checkasm/checkasm.c
+
+checkasm_checkasm_CPPFLAGS = -I$(top_srcdir)/libass
+checkasm_checkasm_LDADD = libass/libass.la
+checkasm_checkasm_LDFLAGS = $(AM_LDFLAGS) -static
+
+if X86
+checkasm_checkasm_SOURCES += checkasm/x86/checkasm.asm
+endif
+if AARCH64
+checkasm_checkasm_SOURCES += checkasm/arm/checkasm_64.S
+endif
+
+run-checkasm: checkasm/checkasm$(EXEEXT)
+	checkasm/checkasm$(EXEEXT)
+
+run-checkasm-bench: checkasm/checkasm$(EXEEXT)
+	checkasm/checkasm$(EXEEXT) --bench
+
+TESTS += checkasm/checkasm$(EXEEXT)
+
+bench: run-checkasm-bench
+
+endif
```

Comment:
Since this is the only reference to `bench`, this means if libass was configured without checkasm support there’s no `bench` rule at all. I’m undecided on whether that’s good or bad.

A minor nit regarding the whole `if ENABLE_CHECKASM` block, but feel free to ignore: 
checkasm sources end up in the dist tarball either way, but it would be more consistent with our other conditional programs, to only enclose those lines into the conditional block which register the program. I.e. something like this:
```automake
if ENABLE_CHECKASM
check_PROGRAMS += checkasm/checkasm
TESTS += checkasm/checkasm$(EXEEXT)
# Not sure if the bench target should also be declared (without any operation) outside
bench: run-checkasm-bench
endif

checkasm_checkasm_SOURCES = ...
```

Reply:
Done.